src/navigation/vertical-navigation-directive.js

```diff
@@ -43,8 +43,8 @@
  * @example
```

Comment:
What are all these changes to vert. nav?  Doesn't seem to have anything to do with removing jQuery or pf-select?